src/tools/roc_recv/cmdline.ggo

```diff
@@ -80,6 +81,9 @@ DRIVER is the type of the output file or device, e.g.:
 PORT is a triplet PROTOCOL:IPADDR:PORTNUM, e.g.:
   rtp+rs8m::10001; rtp+rs8m:127.0.0.1:10001; rtp+rs8m:[::1]:10001;
 
+IFACE_IP is an IP address in one of the following formats:
+  0.0.0.0; 192.168.0.3; [::], [2001:db8::1];
```

Comment:
`IFACE_IP is an IP address in one of the following formats:` -> `IFACE_IP is an IPv4 or IPv6 address, e.g.:`

(because the next line is not list of *formats*; it's a list of some examples)

Reply:
Agree. Fixed